[2024-04-22T00:00:39.818+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-04-21T00:00:00+00:00 [queued]>
[2024-04-22T00:00:39.823+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-04-21T00:00:00+00:00 [queued]>
[2024-04-22T00:00:39.823+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-04-22T00:00:39.832+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): normalize_numerical_features> on 2024-04-21 00:00:00+00:00
[2024-04-22T00:00:39.836+0000] {standard_task_runner.py:60} INFO - Started process 4001 to run task
[2024-04-22T00:00:39.839+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'process_etl_heart_data', 'normalize_numerical_features', 'scheduled__2024-04-21T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/etl_process.py', '--cfg-path', '/tmp/tmp83a7hu56']
[2024-04-22T00:00:39.840+0000] {standard_task_runner.py:88} INFO - Job 20: Subtask normalize_numerical_features
[2024-04-22T00:00:39.877+0000] {task_command.py:423} INFO - Running <TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-04-21T00:00:00+00:00 [running]> on host 3ab9e2e1209e
[2024-04-22T00:00:40.068+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Facundo Adrian Lucianna' AIRFLOW_CTX_DAG_ID='process_etl_heart_data' AIRFLOW_CTX_TASK_ID='normalize_numerical_features' AIRFLOW_CTX_EXECUTION_DATE='2024-04-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-21T00:00:00+00:00'
[2024-04-22T00:00:40.070+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venv_3jctnnp --system-site-packages --python=python
[2024-04-22T00:00:40.079+0000] {process_utils.py:186} INFO - Output:
[2024-04-22T00:00:40.511+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 148ms
[2024-04-22T00:00:40.512+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venv_3jctnnp, clear=False, no_vcs_ignore=False, global=True)
[2024-04-22T00:00:40.512+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2024-04-22T00:00:40.512+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.3.1, setuptools==69.0.2, wheel==0.42.0
[2024-04-22T00:00:40.513+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-04-22T00:00:40.536+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv_3jctnnp/bin/pip install -r /tmp/venv_3jctnnp/requirements.txt
[2024-04-22T00:00:40.544+0000] {process_utils.py:186} INFO - Output:
[2024-04-22T00:00:41.567+0000] {process_utils.py:190} INFO - Collecting awswrangler==3.6.0 (from -r /tmp/venv_3jctnnp/requirements.txt (line 1))
[2024-04-22T00:00:41.569+0000] {process_utils.py:190} INFO -   Using cached awswrangler-3.6.0-py3-none-any.whl.metadata (23 kB)
[2024-04-22T00:00:41.620+0000] {process_utils.py:190} INFO - Collecting mlflow==2.10.2 (from -r /tmp/venv_3jctnnp/requirements.txt (line 2))
[2024-04-22T00:00:41.621+0000] {process_utils.py:190} INFO -   Using cached mlflow-2.10.2-py3-none-any.whl.metadata (13 kB)
[2024-04-22T00:00:41.623+0000] {process_utils.py:190} INFO - Requirement already satisfied: scikit-learn==1.3.2 in /home/***/.local/lib/python3.8/site-packages (from -r /tmp/venv_3jctnnp/requirements.txt (line 3)) (1.3.2)
[2024-04-22T00:00:41.654+0000] {process_utils.py:190} INFO - Requirement already satisfied: boto3<2.0.0,>=1.20.32 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (1.34.88)
[2024-04-22T00:00:41.654+0000] {process_utils.py:190} INFO - Requirement already satisfied: botocore<2.0.0,>=1.23.32 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (1.34.88)
[2024-04-22T00:00:41.655+0000] {process_utils.py:190} INFO - Requirement already satisfied: numpy<2.0,>=1.18 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (1.24.4)
[2024-04-22T00:00:41.656+0000] {process_utils.py:190} INFO - Requirement already satisfied: packaging<24.0,>=21.1 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (23.2)
[2024-04-22T00:00:41.656+0000] {process_utils.py:190} INFO - Requirement already satisfied: pandas<2.1.0,>=1.2.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (2.0.3)
[2024-04-22T00:00:41.657+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyarrow>=7.0.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (14.0.2)
[2024-04-22T00:00:41.657+0000] {process_utils.py:190} INFO - Requirement already satisfied: typing-extensions<5.0.0,>=4.4.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (4.9.0)
[2024-04-22T00:00:41.692+0000] {process_utils.py:190} INFO - Requirement already satisfied: click<9,>=7.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (8.1.7)
[2024-04-22T00:00:41.693+0000] {process_utils.py:190} INFO - Requirement already satisfied: cloudpickle<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.0.0)
[2024-04-22T00:00:41.693+0000] {process_utils.py:190} INFO - Requirement already satisfied: entrypoints<1 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (0.4)
[2024-04-22T00:00:41.694+0000] {process_utils.py:190} INFO - Requirement already satisfied: gitpython<4,>=2.1.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.1.43)
[2024-04-22T00:00:41.695+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyyaml<7,>=5.1 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (6.0.1)
[2024-04-22T00:00:41.696+0000] {process_utils.py:190} INFO - Requirement already satisfied: protobuf<5,>=3.12.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (4.25.2)
[2024-04-22T00:00:41.697+0000] {process_utils.py:190} INFO - Requirement already satisfied: pytz<2024 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (2023.3.post1)
[2024-04-22T00:00:41.698+0000] {process_utils.py:190} INFO - Requirement already satisfied: requests<3,>=2.17.3 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (2.31.0)
[2024-04-22T00:00:41.698+0000] {process_utils.py:190} INFO - Requirement already satisfied: importlib-metadata!=4.7.0,<8,>=3.7.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (6.11.0)
[2024-04-22T00:00:41.699+0000] {process_utils.py:190} INFO - Requirement already satisfied: sqlparse<1,>=0.4.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (0.4.4)
[2024-04-22T00:00:41.700+0000] {process_utils.py:190} INFO - Requirement already satisfied: alembic!=1.10.0,<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (1.13.1)
[2024-04-22T00:00:41.700+0000] {process_utils.py:190} INFO - Requirement already satisfied: docker<8,>=4.0.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (7.0.0)
[2024-04-22T00:00:41.701+0000] {process_utils.py:190} INFO - Requirement already satisfied: Flask<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (2.2.5)
[2024-04-22T00:00:41.701+0000] {process_utils.py:190} INFO - Requirement already satisfied: scipy<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (1.10.1)
[2024-04-22T00:00:41.702+0000] {process_utils.py:190} INFO - Requirement already satisfied: querystring-parser<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (1.2.4)
[2024-04-22T00:00:41.703+0000] {process_utils.py:190} INFO - Requirement already satisfied: sqlalchemy<3,>=1.4.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (1.4.51)
[2024-04-22T00:00:41.703+0000] {process_utils.py:190} INFO - Requirement already satisfied: markdown<4,>=3.3 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.5.2)
[2024-04-22T00:00:41.704+0000] {process_utils.py:190} INFO - Requirement already satisfied: matplotlib<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.7.5)
[2024-04-22T00:00:41.705+0000] {process_utils.py:190} INFO - Requirement already satisfied: gunicorn<22 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (21.2.0)
[2024-04-22T00:00:41.706+0000] {process_utils.py:190} INFO - Requirement already satisfied: Jinja2<4,>=2.11 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.1.3)
[2024-04-22T00:00:41.727+0000] {process_utils.py:190} INFO - Requirement already satisfied: joblib>=1.1.1 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venv_3jctnnp/requirements.txt (line 3)) (1.4.0)
[2024-04-22T00:00:41.727+0000] {process_utils.py:190} INFO - Requirement already satisfied: threadpoolctl>=2.0.0 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venv_3jctnnp/requirements.txt (line 3)) (3.4.0)
[2024-04-22T00:00:41.733+0000] {process_utils.py:190} INFO - Requirement already satisfied: Mako in /home/***/.local/lib/python3.8/site-packages (from alembic!=1.10.0,<2->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (1.3.0)
[2024-04-22T00:00:41.734+0000] {process_utils.py:190} INFO - Requirement already satisfied: importlib-resources in /home/***/.local/lib/python3.8/site-packages (from alembic!=1.10.0,<2->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (5.13.0)
[2024-04-22T00:00:41.737+0000] {process_utils.py:190} INFO - Requirement already satisfied: jmespath<2.0.0,>=0.7.1 in /home/***/.local/lib/python3.8/site-packages (from boto3<2.0.0,>=1.20.32->awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (0.10.0)
[2024-04-22T00:00:41.738+0000] {process_utils.py:190} INFO - Requirement already satisfied: s3transfer<0.11.0,>=0.10.0 in /home/***/.local/lib/python3.8/site-packages (from boto3<2.0.0,>=1.20.32->awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (0.10.1)
[2024-04-22T00:00:41.743+0000] {process_utils.py:190} INFO - Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in /home/***/.local/lib/python3.8/site-packages (from botocore<2.0.0,>=1.23.32->awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (2.8.2)
[2024-04-22T00:00:41.744+0000] {process_utils.py:190} INFO - Requirement already satisfied: urllib3<1.27,>=1.25.4 in /home/***/.local/lib/python3.8/site-packages (from botocore<2.0.0,>=1.23.32->awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (1.26.18)
[2024-04-22T00:00:41.757+0000] {process_utils.py:190} INFO - Requirement already satisfied: Werkzeug>=2.2.2 in /home/***/.local/lib/python3.8/site-packages (from Flask<4->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (2.2.3)
[2024-04-22T00:00:41.758+0000] {process_utils.py:190} INFO - Requirement already satisfied: itsdangerous>=2.0 in /home/***/.local/lib/python3.8/site-packages (from Flask<4->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (2.1.2)
[2024-04-22T00:00:41.773+0000] {process_utils.py:190} INFO - Requirement already satisfied: gitdb<5,>=4.0.1 in /home/***/.local/lib/python3.8/site-packages (from gitpython<4,>=2.1.0->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (4.0.11)
[2024-04-22T00:00:41.805+0000] {process_utils.py:190} INFO - Requirement already satisfied: zipp>=0.5 in /home/***/.local/lib/python3.8/site-packages (from importlib-metadata!=4.7.0,<8,>=3.7.0->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.17.0)
[2024-04-22T00:00:41.808+0000] {process_utils.py:190} INFO - Requirement already satisfied: MarkupSafe>=2.0 in /home/***/.local/lib/python3.8/site-packages (from Jinja2<4,>=2.11->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (2.1.3)
[2024-04-22T00:00:41.825+0000] {process_utils.py:190} INFO - Requirement already satisfied: contourpy>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (1.1.1)
[2024-04-22T00:00:41.825+0000] {process_utils.py:190} INFO - Requirement already satisfied: cycler>=0.10 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (0.12.1)
[2024-04-22T00:00:41.826+0000] {process_utils.py:190} INFO - Requirement already satisfied: fonttools>=4.22.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (4.51.0)
[2024-04-22T00:00:41.826+0000] {process_utils.py:190} INFO - Requirement already satisfied: kiwisolver>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (1.4.5)
[2024-04-22T00:00:41.828+0000] {process_utils.py:190} INFO - Requirement already satisfied: pillow>=6.2.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (10.3.0)
[2024-04-22T00:00:41.828+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyparsing>=2.3.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.1.1)
[2024-04-22T00:00:41.889+0000] {process_utils.py:190} INFO - Requirement already satisfied: tzdata>=2022.1 in /home/***/.local/lib/python3.8/site-packages (from pandas<2.1.0,>=1.2.0->awswrangler==3.6.0->-r /tmp/venv_3jctnnp/requirements.txt (line 1)) (2023.4)
[2024-04-22T00:00:41.902+0000] {process_utils.py:190} INFO - Requirement already satisfied: six in /home/***/.local/lib/python3.8/site-packages (from querystring-parser<2->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (1.16.0)
[2024-04-22T00:00:41.908+0000] {process_utils.py:190} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.3.2)
[2024-04-22T00:00:41.909+0000] {process_utils.py:190} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.6)
[2024-04-22T00:00:41.909+0000] {process_utils.py:190} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (2023.11.17)
[2024-04-22T00:00:41.980+0000] {process_utils.py:190} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from sqlalchemy<3,>=1.4.0->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (3.0.3)
[2024-04-22T00:00:42.049+0000] {process_utils.py:190} INFO - Requirement already satisfied: smmap<6,>=3.0.1 in /home/***/.local/lib/python3.8/site-packages (from gitdb<5,>=4.0.1->gitpython<4,>=2.1.0->mlflow==2.10.2->-r /tmp/venv_3jctnnp/requirements.txt (line 2)) (5.0.1)
[2024-04-22T00:00:42.165+0000] {process_utils.py:190} INFO - Using cached awswrangler-3.6.0-py3-none-any.whl (403 kB)
[2024-04-22T00:00:42.167+0000] {process_utils.py:190} INFO - Using cached mlflow-2.10.2-py3-none-any.whl (19.5 MB)
[2024-04-22T00:00:43.989+0000] {process_utils.py:190} INFO - Installing collected packages: mlflow, awswrangler
[2024-04-22T00:00:43.990+0000] {process_utils.py:190} INFO -   Attempting uninstall: mlflow
[2024-04-22T00:00:43.991+0000] {process_utils.py:190} INFO -     Found existing installation: mlflow 2.12.1
[2024-04-22T00:00:43.991+0000] {process_utils.py:190} INFO -     Not uninstalling mlflow at /home/***/.local/lib/python3.8/site-packages, outside environment /tmp/venv_3jctnnp
[2024-04-22T00:00:43.992+0000] {process_utils.py:190} INFO -     Can't uninstall 'mlflow'. No files were found to uninstall.
[2024-04-22T00:00:45.819+0000] {process_utils.py:190} INFO - Successfully installed awswrangler-3.6.0 mlflow-2.10.2
[2024-04-22T00:00:46.160+0000] {process_utils.py:190} INFO - 
[2024-04-22T00:00:46.160+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 23.3.1 -> 24.0
[2024-04-22T00:00:46.160+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2024-04-22T00:00:46.267+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv_3jctnnp/bin/python /tmp/venv-callpbt6hhls/script.py /tmp/venv-callpbt6hhls/script.in /tmp/venv-callpbt6hhls/script.out /tmp/venv-callpbt6hhls/string_args.txt /tmp/venv-callpbt6hhls/termination.log
[2024-04-22T00:00:46.273+0000] {process_utils.py:186} INFO - Output:
[2024-04-22T00:00:47.937+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:47.936+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-22T00:00:47.977+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:47.977+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-22T00:00:48.006+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:48.006+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-22T00:00:48.140+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:48.127+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-22T00:00:48.159+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:48.158+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-22T00:00:48.190+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:48.190+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-22T00:00:48.212+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:48.212+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-22T00:00:48.246+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:48.245+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-22T00:00:48.262+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:48.262+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-22T00:00:48.298+0000] {process_utils.py:190} INFO - [[34m2024-04-22T00:00:48.298+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-22T00:00:49.540+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-04-22T00:00:49.549+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=process_etl_heart_data, task_id=normalize_numerical_features, execution_date=20240421T000000, start_date=20240422T000039, end_date=20240422T000049
[2024-04-22T00:00:49.565+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-22T00:00:49.576+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
